src/lib.rs

```diff
@@ -179,7 +191,7 @@ enum FieldSpec {
 }
 
 #[derive(Debug, PartialEq, Clone, Copy)]
-enum OperandSpec {
+pub enum OperandSpec {
```

Comment:
Why make this one public? IIRC the convention has been to leave [`OperandSpec`](https://github.com/DrChat/yaxpeax-x86/blob/8f0ef83ff866df7ca941ae7b2adf105ce2e6d3a6/src/long_mode/mod.rs#L241-L242) private and expose a user-friendly `Operand` instead.

Reply:
I actually don't remember why I did this -- in the current code I'm just using `Opcode` and `Operand`. I'll revert these two changes.